cache/src/main/java/com/uber/buckcache/resources/buckcache/BuckCacheResource.java

```diff
@@ -77,6 +78,29 @@ public String getSummary() throws Exception {
     }
   }
 
+  /**
+   * Used for client side to measure download speed.
+   * @param size
+   * @return
+   * @throws Exception
+   */
+  @GET
+  @Path("dummy")
+  @Produces(MediaType.APPLICATION_OCTET_STREAM)
+  public String getDummyArtifact() throws Exception {
+    Random rand = new Random();
+    rand.setSeed(System.currentTimeMillis());
+    try {
+      // 100Kb
+      byte[] bytes = new byte[100000];
+      rand.nextBytes(bytes);
+      return new String(bytes);
+    } catch (Exception e) {
+      e.printStackTrace();
+      return e.getMessage();
```

Comment:
It might be better to just raise an exception (which will probably result in a server error) instead of returning an error-string with a successful response.